### PR TITLE
Add a team counter for folders

### PR DESCRIPTION
**Pull request by Spandamn**
Suggestion from [this thread](https://www.smogon.com/forums/threads/team-counter.3696070/)
image
image

### CODE REVIEW

**Comment by Spandamn** [comment]
What the fuck, first why do u hate the arrow function

**Comment by Spandamn** [comment]
someone suggest an alternative

File: src/panel-teambuilder.tsx [hunk 1]
```diff
@@ -234,9 +234,9 @@ class TeambuilderPanel extends PSRoomPanel<TeambuilderRoom> {
 				: filterFolder === '' ?
 					<h2><i class="fa fa-folder-open-o"></i> Teams not in any folders</h2>
 				: filterFormat ?
-					<h2><i class="fa fa-folder-open-o"></i> {filterFormat}</h2>
+					<h2><i class="fa fa-folder-open-o"></i> {`${filterFormat} (${teams.length})`}</h2>
```
Comment (KrisXV) [inline src/panel-teambuilder.tsx:237]:
```suggestion
					<h2><i class="fa fa-folder-open-o"></i> {filterFormat} ({teams.length})</h2>
```

File: src/panel-teambuilder.tsx [hunk 2]
```diff
@@ -234,9 +234,9 @@ class TeambuilderPanel extends PSRoomPanel<TeambuilderRoom> {
...
 				:
-					<h2>All Teams</h2>
+					<h2>All Teams {`${teams.length}`}</h2>
```
Comment (KrisXV) [inline src/panel-teambuilder.tsx:239]:
```suggestion
					<h2>All Teams ({teams.length})</h2>
```

File: js/client-teambuilder.js [hunk 3]
```diff
@@ -347,7 +347,7 @@
 				buf += '<h2>Hi</h2>';
 				buf += '<p>Did you have a good day?</p>';
 				buf += '<p><button class="button" name="greeting" value="Y"><i class="fa fa-smile-o"></i> Yes, my day was pretty good</button> <button class="button" name="greeting" value="N"><i class="fa fa-frown-o"></i> No, it wasn\'t great</button></p>';
-				buf += '<h2>All teams</h2>';
+				buf += '<h2>All teams (' + teams.length + ')</h2>';
```
Comment (Zarel) [inline js/client-teambuilder.js:350]: This should be in `<small>`, and probably `<small>` should be styled not to be bold. Otherwise, good idea!

**Comment by Spandamn** [comment]
image
 Your suggestion made it look much better, how do i remove the bold though? Take it out of h2?

**Comment by Spandamn** [comment]
image
Seems like thats not gonna work... wonder how one would keep em together

**Comment by Zarel** [comment]
> Your suggestion made it look much better, how do i remove the bold though? Take it out of h2?

CSS... you seem to have figured it out. Ideally this should go in CSS files, but whatever, we can do that refactor later. Thanks!

**Pull request merged**